assets/app/styles/_components.less

```diff
@@ -0,0 +1,269 @@
+      }
+      &:before {
+        content: '\f111';
+        font: normal normal normal 7px/1 FontAwesome;
+        display: inline-block;
+        color: #fff;
+        position: absolute;
+        left: 1px;
+        top:1px;
+      }
+    }
+    .osc-object-active {
+      .connector {
+        i {
+          color: @osc-object-active-color;
+        }
+      }
+    }
+  }
+  .components-panel {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    position: relative;
+    .osc-object-highlight();
+    .component-label {
+      text-transform: uppercase;
+      font-size: 11px;
+      color: @gray-light;
+      + span {
+        margin-left: 5px;
+      }
+    }
+    .component-block {
+      .flex-display(@display: flex);
+      .flex-direction(@direction: row);
+      .flex-wrap(@wrap: wrap);
+      &.pod-template-container {
+        margin-top: 5px;
+      }
+      &.component-meta {
+        margin: 4px 0 6px;
+      }
+      .component {
+        .flex(@columns: 1);
+        h2,h3 {
+          margin: 0 0 5px;
+        }
+        &.meta-data {
+          .flex(@columns: 1);
+          text-align: right;
+          .align-self(@align: center)
+        }
+        &.pod-template-block {
+          text-align: left;
+          display: inline-block;
+          .flex(@columns: 1 0 auto);
+          max-width: 450px;
+          border-radius: 3px;
+          margin-bottom: 10px;
+          &:nth-of-type(odd) {
+            margin-right: 10px;
+          }
+          .pod-template {
+            border: 1px dotted #ddd;
+            border-radius: @border-radius-md;
+            padding: 5px 10px;
+            h4 {
+              margin: 5px 0;
+            }
+            .fa {
+              margin-right: 5px;
+            }
+          }
+          .pod-template-ports {
+            .font-icon-block {
+              margin-right: 8px;
+            }
+            .port-number {
+              display: inline-block;
+            }
+          }
+        }
+      }
+    }
+    &.service {
+      border-left-width: 0;
+      margin-bottom: 6px;
+      padding: 6px 15px 5px;
+      .osc-object-highlight();
+      &.none {
+        background-color: transparent;
+      }
+    }
+    &.deployment-block {
+      margin: 0 0 0 6px;
+      padding: 5px 15px;
+      &.none {
+        background-color: transparent;
+        &:after {
+          background-color: #ccc;
+          top: 50%;
+          content: "";
+          display: block;
+          height: 1px;
+          left: -10px;
+          position: absolute;
+          width: 9px;
+        }
+      }
+    }
+  }
+}
+
+.builds-block {
+  margin-left: 5px;
+  .builds {
+    margin: 0;
+    overflow: hidden;
+    .build {
+      border-radius: @border-radius-md;
+      padding: 5px;
+      position: relative;
+      margin: 10px 0;
+      text-align: left;
+      &:first-child {
+        margin-top: 0;
+      }
+      &:after {
+        background-color: @connector-color;
```

Comment:
Nit: We no longer need the separator under the build message. (Although we might keep it when there are multiple builds.)

![openshift_management_console](https://cloud.githubusercontent.com/assets/1167259/7726396/c114e760-fecf-11e4-8089-ca543d4e0c1b.png)